sky/skylet/job_lib.py

```diff
@@ -212,17 +217,17 @@ def _get_jobs(self) -> List:
 }
 
 _RAY_TO_JOB_STATUS_MAP = {
-    # These are intentionally set to one status before, because:
+    # These are intentionally set this way, because:
     # 1. when the ray status indicates the job is PENDING the generated
-    # python program should not be started yet, i.e. the job should be INIT.
+    # python program has left the job queue and is now PENDING
```

Comment:
What does it mean by `left the job queue`? Does it mean `the generated python program has left the skypilot pending queue and submitted with ray job submit`?